backstop-non-responsive.json

```
@@ -1082,7 +1082,13 @@
         ".test-label-optional-sm",
         ".test-label-optional-md",
         ".test-label-optional-lg",
-        ".test-label-optional-xl"
+        ".test-label-optional-xl",
+        ".test-label-info-icon-message",
+        ".test-label-info-icon-message-xs",
+        ".test-label-info-icon-message-sm",
+        ".test-label-info-icon-message-md",
+        ".test-label-info-icon-message-lg",
+        ".test-label-info-icon-message-xl"
```

The first selector includes the ones below, please remove the others and the pics as well.

```suggestion
        ".test-label-info-icon-message"
```